src/client/java/teammates/client/scripts/sql/VerifyCourseEntityCounts.java

```diff
@@ -72,4 +79,17 @@ protected void doOperation() {
             printEntityVerification(objectifyClass.getSimpleName(), objectifyEntityCount, postgresEntityCount);
         }
     }
+
+    private void verifyNewEntities() {
+        List<CourseStudent> students = ofy().load().type(CourseStudent.class).order("courseId").list();
+        verifySectionEntities(students);
+    }
+
+    private void verifySectionEntities(List<CourseStudent> students) {
+        int objectifyEntityCount = (int) students.stream().map(stu -> stu.getSectionName() + stu.getCourseId())
+                .distinct().count();
```

Comment:
There's 1.7m students in the database, this line may potentially cause an Out Of Memory error. But the requirements will change after today's meeting, so it'll probably work with the new design